### PR TITLE
add script to check duplicate import in go file

### PR DESCRIPTION
**Pull request by stewart-yu**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
add script to check duplicate import in go file

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
```release-note
NONE
```

### CODE REVIEW

**Comment by roycaihw** [comment]
do the staticcheck https://github.com/kubernetes/kubernetes/pull/81189 or golint cover this already?

File: hack/verify-imports.sh [hunk 1]
```diff
@@ -39,3 +39,32 @@ if [[ ! -x "$importverifier" ]]; then
...
+        -o -wholename './hack' \
+        -o -wholename './logo' \
+        -o -wholename './third_party' \
+        -o -wholename './translations' \
+        -o -wholename '*/vendor/*' \
+      \) -prune \
+    \) -name '*.go'
+}
+
+find_files | xargs dedupimport -d -i
```
Comment (roycaihw) [inline hack/verify-imports.sh]: does this line only check duplication, or does it fix the duplication as well?

Comment (stewart-yu) [reply hack/verify-imports.sh]: yes

**Comment by stewart-yu** [comment]
> @stewart-yu this seemed like a useful PR. any plans to update it?

I'm have trouble in using `VM` recently,  and will fix this later
If any guys want to help, that's ok for me

**Pull request closed without merging**